config/puma.rb

```diff
@@ -20,6 +20,9 @@
 #
 environment env
 
+# Specifies the `pidfile` that Puma will use.
+pidfile ENV.fetch("PIDFILE") { "tmp/pids/server.pid" }
```

Comment:
Unrelated to the change, what is “Barnes” required at the top?

Reply:
It's used for heroku's advanced analytics